### PR TITLE
Feat: move value and transformer to general tab

### PR DESCRIPTION
**Pull request by thibault-barrat**
image

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -117,7 +117,7 @@ ifeq "$(DISABLE_E2E_TESTS)" "true"
 else
 	$(MAKE) test-e2e-start-dockers
 	npx cypress install
-	./bin/wait-for -t 30 localhost:3000 -- npx cypress run --browser chrome || (\
+	./bin/wait-for -t 30 localhost:3000 -- npx cypress open --browser chrome || (\
```
Comment (arimet) [inline Makefile]: run

File: src/app/js/fields/FieldDisplay.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-import FormCheckboxField from '../lib/components/FormCheckboxField';
+import FormSwitchField from '../lib/components/FormSwitchField';
 import FieldInput from '../lib/components/FieldInput';
 
 const FieldDisplayInput = () => (
     <FieldInput
         className="display"
         name="display"
-        component={FormCheckboxField}
+        component={FormSwitchField}
```
Comment (arimet) [inline src/app/js/fields/FieldDisplay.js:10]: What's for ?

File: src/app/js/fields/transformers/TransformerUpsertDialog.js [hunk 3]
```diff
@@ -94,7 +95,7 @@ const TransformerUpsertDialog = ({
                 <Box display={'flex'} flexDirection="column">
                     <Autocomplete
                         aria-label={polyglot.t('select_an_operation')}
-                        value={transformer.operation}
+                        value={transformer.operation || ''}
```
Comment (arimet) [inline src/app/js/fields/transformers/TransformerUpsertDialog.js:98]: Empty is not the same as undefined,  does it change something ?

File: src/app/js/fields/wizard/TabGeneral.js [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+import { FIELD_FORM_NAME } from '..';
+
+export const isSubresourceTransformation = transformers => {
+    const operations = transformers.map(t => t.operation).join('|');
+    return (
+        operations === 'COLUMN|PARSE|GET|STRING|REPLACE_REGEX|MD5|REPLACE_REGEX'
+    );
+};
+
+export const renderTransformer = (
```
Comment (arimet) [inline src/app/js/fields/wizard/TabGeneral.js:27]: A function that return a function.
Could be directly a component

File: src/app/js/fields/wizard/index.js [hunk 5]
```diff
@@ -121,27 +119,16 @@ const FieldEditionWizardComponent = ({
...
             id: 'tab-identity',
-            component: <TabIdentity field={field} />,
-        },
-        {
-            label: 'field_wizard_tab_value',
-            id: 'tab-value',
             component: (
-                <TabValue
+                <TabGeneral
+                    field={field}
```
Comment (arimet) [inline src/app/js/fields/wizard/index.js]: Wich field ? From that naming, i can't understand what is the point of this prop.

**Review by arimet — CHANGES_REQUESTED** [review]
I'm going to be boring but I think that all the fields of the same "Tab" should have the same design. 

Title and internal name should be changed

File: src/app/js/fields/transformers/TransformerUpsertDialog.js (hunk 3 above)
Comment (thibault-barrat) [reply src/app/js/fields/transformers/TransformerUpsertDialog.js:98]: It's to have a controlled field

File: src/app/js/fields/wizard/TabGeneral.js (hunk 4 above)
Comment (thibault-barrat) [reply src/app/js/fields/wizard/TabGeneral.js:27]: it's because redux form Field does not accept JSX component with props

**Review by arimet — APPROVED** [review]

**Pull request merged**